# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/audioipc-server/.cargo-checksum.json
# Commit: 634412ce2943
# Full Hash: 634412ce29430557eec6d0c0313f1908e61ad13d
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2021-08-11 03:46:10
# Regressor Bug: 1724141
# File Overlap Count: 6
# Description:
#   Bug 1724141 - Update audioipc to 8bb1a227.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D121826
# ==============================================================================

diff -r 722284bed619 -r 634412ce2943 third_party/rust/audioipc-server/.cargo-checksum.json
--- a/third_party/rust/audioipc-server/.cargo-checksum.json	Tue Aug 10 20:38:51 2021 +0000
+++ b/third_party/rust/audioipc-server/.cargo-checksum.json	Tue Aug 10 20:52:03 2021 +0000
@@ -1,1 +1,1 @@
-{"files":{"Cargo.toml":"6329179497fb654bec0dea9f3642056309de3fa37d4042a48d18224e7b4742d3","cbindgen.toml":"bd89c5a9f52395b1c703ff04d1c0019dc3c92b691d571ae503c4b85753a44a39","src/lib.rs":"2e56bed9ea0012222bab33680a926b3761c66f455253baf09bd0166ce38237f4","src/server.rs":"7404bfdb69c6b830937dcc81c7996d638737ea4010ee255eb9895ff938b1a198"},"package":null}
\ No newline at end of file
+{"files":{"Cargo.toml":"6329179497fb654bec0dea9f3642056309de3fa37d4042a48d18224e7b4742d3","cbindgen.toml":"bd89c5a9f52395b1c703ff04d1c0019dc3c92b691d571ae503c4b85753a44a39","src/lib.rs":"5da72fb551ea8a15df71e9d9e260e20d5c9571194b5d523bc39b257b2393f60f","src/server.rs":"045e54a1a015c75f9fa6c26ed426d227cfa653c4d851c16c862b9ccd57829393"},"package":null}
\ No newline at end of file